back/src/Guide/Main.hs

```diff
@@ -163,6 +175,26 @@ mainWith config@Config{..} = withLogger config $ \logger -> do
   forever (threadDelay (1000000 * 60))
     `finally` cancel workAsync
 
+-- | Load database and exit.
+dryRun :: Config -> IO ()
+dryRun config = withLogger config $ \logger -> do
+  db :: DB <- openLocalStateFrom "state/" (error "couldn't load state")
+  logDebugIO logger "loaded the database successfully"
+  closeAcidState db
+  exitSuccess
+
+-- | Load PublicDB, create base on it and exit.
+loadPublic :: String -> Config -> IO ()
```

Comment:
```suggestion
loadPublic :: FilePath -> Config -> IO ()
```

I also think it should take `Config` as the first argument rather than the second.